rfcs/text/0008-threat-intel.md

```diff
@@ -87,171 +83,139 @@ Stage 3: Add or update all remaining field definitions. The list should now be e
 
-There are two primary uses for these fields.
+While there are two primary uses for these fields, this RFC deals primarily with the first: ingestion/storage of threat intelligence.
 
-1. **Storing threat intelligence as an event document in threat index(s).**
+**Storing threat intelligence as an event document in threat index(s).**
 
-    Threat intelligence data will be collected from multiple sources stored in threat indices. The ECS fields proposed here will be used to structure the documents collected from various sources.
+Threat intelligence data will be collected from multiple sources stored in threat indices. The ECS fields proposed here will be used to structure the documents collected from various sources.
 
-**Example**
+**Examples**
+
+Network Example
 ```json5
 {
+    // Metadata about the indicator event
     "@timestamp": "2019-08-10T11:09:23.000Z",
-    "event.kind": "enrichment",
-    "event.category": "threat",
-    "event.type": "indicator",
-    "event.provider": "Abuse.ch",
-    "event.reference": "https://feodotracker.abuse.ch",
-    "event.dataset": "threatintel.abusemalware",
-    "event.module": "threatintel",
-
-    // The top-level file object here allows expressing multiple indicators for a single file object
-    "file.hash.sha256": "0c415dd718e3b3728707d579cf8214f54c2942e964975a5f925e0b82fea644b4",
-    "file.hash.md5": "1eee2bf3f56d8abed72da2bc523e7431",
-    "file.size": 656896,
-    "file.name": "invoice.doc",
-
-    // The indicator prefix here gives context of the indicators
-    "indicator.marking.tlp": "WHITE",
-    "indicator.time_first_seen": "2020-10-01",
-    "indicator.time_last_seen": "2020-11-01",
-    "indicator.sightings": "4",
-    /* It's possible to have multiple related indicators in a given document,
-       e.g. sha256, sha1, ssdeep, etc. If that's the case this should be an array
-       of types (i.e. [sha1, sha256, ssdeep]) */
-    "indicator.type": ["sha256", "md5", "file_name", "file_size"],
-    "indicator.description": "file last associated with delivering Angler EK",
-
-    // Filebeats and other fields, not part of ECS proposal
-    "fileset.name": "abusemalware",
-    "input.type": "log",
-    "log.offset": 0,
+    "event": {
+        "kind": "enrichment",
+        "category": "threat",
+        "type": "indicator",
+        "reference": "https://urlhaus.abuse.ch",
+        "severity": 7,
+        "risk_score": 10,
+        "original": "2020-10-29 19:16:38"
+    },
+
+    // Metadata about the indicator data
+    "threat.indicator": {
+        "first_seen": "2020-10-01",
```

Comment:
@peasead it's probably outside the scope of this RFC, but I wanted to mention that the need for an `updated_at` field has been requested in the context of indicator indexes, in order to be able to query the most relevant indicators by sorting on said field. I think that this goes against our current idea of indicators as timeseries data, but the underlying problem can be expressed as three events, in chronological order:

1. Domain indicator A is active
2. Domain indicator B is active
3. Domain indicator A is inactive

If the analyst is only concerned with active domains, then a naive query may include both active and inactive versions of indicator A if scoping simply on `@timestamp` or `event.ingested`.

Reply:
I think we should add it now.

I'll do that.